CHANGELOG.next.asciidoc

```diff
@@ -199,6 +200,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add `xpack.enabled` support for Enterprise Search module. {pull}29871[29871]
 - Add gcp firestore metricset. {pull}29918[29918] 
 - Remove strict parsing on RabbitMQ module {pull}30090[30090]
+- Add gcp firestore metricset. {pull}29918[29918]
```

Comment:
Please move this change to a separate PR. AFAIK editing multiple lines can cause issues when backporting and preparing changelogs before release.

Reply:
Hmm, somehow rebase on master added this, weird.